docs/tutorials/embedding/AdjacencySpectralEmbed.ipynb

```diff
@@ -120,10 +254,10 @@
    "source": [
     "palette = {'Right':(0,0.7,0.2),\n",
     "           'Wrong':(0.8,0.1,0.1)}\n",
+    "\n",
     "for insularity in np.linspace(0.5, 0.625, 4):\n",
```

Comment:
side note, I think this block of code is kind of inappropriately confusing for a tutorial. I didn't refactor it since it'd take some time and it wasn't really in-scope, but I think maybe we should?

Reply:
i don't think you're limited to using the examples from the ASE tutorial just FYI. if you think something else is wrong I'd just make another issue for later. for OOSASE i think this whole for loop idea may be too much